### ISSUE TITLE
Checkout find_package default logging

### ISSUE THREAD

**Issue by lulivi** [rticommunity/rticonnextdds-examples#244]
The problem is that with CMake <=3.14 every logging will be displayed (see [here](https://github.com/rticommunity/rticonnextdds-examples/blob/develop/resources/cmake/FindRTIConnextDDS.cmake#L382-L390)):
```cmake
else()
    macro(connextdds_log_debug message)
        message("DEBUG ${message}")
    endmacro()

    macro(connextdds_log_verbose message)
        message("VERBOSE ${message}")
    endmacro()
endif()
```

**Comment by lulivi** [comment]
@iblancasa @fgaranda Sholuld we add some kind of variable to define what log level we want for CMake <= 3.14?
Maybe something like `-DLOG_LEVEL=<level>` being `<level>` one of 
- `VERBOSE`
- `DEBUG`
What do you think?

**Comment by iblancasa** [comment]
Yes, you need to implement a verbosity level. I though it was implemented when logging capabilities were added.

The name of the CMake variable should be something like `CONNEXTDDS_LOG_LEVEL` or something similar.

**Comment by lulivi** [comment]
Fixed in #252 .
